graylog2-server/src/main/java/org/graylog2/indexer/indices/Indices.java

```diff
@@ -585,4 +596,21 @@ public TimestampStats timestampStatsOfIndex(String index) {
 
         return TimestampStats.create(min, max);
     }
+
+    public static class ESAliasesException extends Exception {
```

Comment:
How about naming it `TooManyAliasTargetsException` (and also making it a standalone class)?